terraform-provider-constellation/examples/full/azure_cluster.tf

```diff
@@ -0,0 +1,111 @@
+terraform {
+  required_providers {
+    constellation = {
+      source  = "edgelesssys/constellation"
+      version = "X.Y.Z"
+    }
+    random = {
+      source  = "hashicorp/random"
+      version = "3.6.0"
+    }
+  }
+}
+
+locals {
+  name                = "constell"
+  version             = "vX.Y.Z"
+  csp                 = "azure"
+  attestation_variant = "azure-sev-snp"
+  location            = "northeurope"
+
+  master_secret      = random_bytes.master_secret.hex
+  master_secret_salt = random_bytes.master_secret_salt.hex
+  measurement_salt   = random_bytes.measurement_salt.hex
+}
+
+resource "random_bytes" "master_secret" {
+  length = 32
+}
+
+resource "random_bytes" "master_secret_salt" {
+  length = 32
+}
+
+resource "random_bytes" "measurement_salt" {
+  length = 32
+}
+
+module "azure_iam" {
+  source                 = "./infrastructure/iam/azure" # TODO: Replace with remote source URL
```

Comment:
Can we add the remote source URLs now, or do we need something else for this?

Reply:
I don't know how the artifact will look like in the actual release, but we can probably make a correct guess on that given that nobody further modifies the uploading code, which should be the case.

Reply:
the URL looks good